src/pages/Admin/AdminProductCategoires.tsx

```diff
@@ -0,0 +1,100 @@
+import AppContainer from '@components/common/container/AppContainer';
+import NavBar from '@components/common/nav/NavBar';
+import TitleNavBar from '@components/common/nav/TitleNavBar';
+import styled from '@emotion/styled';
+import AppButton from '@components/common/button/AppButton';
+import AppInputWithButton from '@components/common/input/AppInputWithButton';
+import { useRef } from 'react';
+import useAdminProducts from '@hooks/admin/useAdminProducts';
+import { useParams } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+import { categoriesAtom } from '@recoils/atoms';
+import DragAndDropContent from '@components/common/content/DragAndDropContent';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  align-items: center;
+  padding-top: 100px;
+  padding-bottom: 100px;
+`;
+
+const CategoriesInputContainer = styled.div`
+  width: 100%;
+  height: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+const CategoriesContentContainer = styled.div`
+  width: 100%;
+  height: 300px;
```

Comment:
```suggestion
  height: 500px;
```

지금은 좀 좁아서 카테고리가 가려지는 갯수가 많은 것 같아요
좀 더 높여서 많은 카테고리가 보이게끔 하시죠